app/src/main/java/org/p2p/wallet/history/interactor/mapper/RpcHistoryTransactionConverter.kt

```diff
@@ -243,6 +243,12 @@ class RpcHistoryTransactionConverter(
             bundle?.fees?.arbiterFee,
             bundle?.fees?.gasFeeInToken
         )
+        val isFree = bundle?.compensationDeclineReason?.isEmpty() ?: false
```

Comment:
isClaimFree?

Reply:
we already know that it is claim =[

Reply:
we are in the parseWormholeReceive method, what should i think of? isReceiveFree? isBundleFree? isDeclineReasonFree? I couldn't understand it with the context provided, sorry
Damn, and which dog did bite you today?

Reply:
isTransactionWasFreeForClaimThatWasDoneInEthereum